cypress/e2e/ui/Overview/dashboard.cy.js

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable no-undef */
+
+describe('Overview > Dashboard Tests', () => {
+  const defaultCards = [];
+  let newCard = '';
+
+  beforeEach(() => {
+    cy.login();
+    cy.menu('Overview', 'Dashboard');
+  });
+
+  it('Dashboard loads correctly', () => {
+    cy.get('#main-menu');
+    cy.get('.card-pf').then((cards) => {
+      const nums = [...Array(cards.length).keys()];
+      nums.forEach((index) => {
+        defaultCards[index] = cards[index].firstChild.innerText;
+      });
+    });
+  });
+
+  it('Default dashboard widgets loaded', () => {
+    cy.get('.card-pf').then((cards) => {
+      const nums = [...Array(cards.length).keys()];
+      nums.forEach((index) => {
+        cy.get(cards[index]).contains(defaultCards[index]);
+      });
+    });
+  });
+
+  it('Reset dashboard back to default widgets', () => {
+    cy.get('#dropdown-custom-2').click().then(() => {
+      cy.get('.scrollable-options').then((list) => {
+        cy.get(list.children()[0]).click();
+      });
+    });
+
+    cy.get('#dropdown-custom-2').click().then(() => {
+      cy.get('.scrollable-options').then((list) => {
+        cy.get(list.children()[1]).click();
+      });
+    });
+
+    cy.wait(5000);
```

Comment:
`cy.wait` should never be needed because the .get should wait until the thing you're getting is ready and visible.  I'm curious why this is needed.

Reply:
As discussed going to merge anyway, and then this can be cleaned up afterwards.